meta/contao/installation-bundle/en.yml

```diff
@@ -0,0 +1,5 @@
+en:
+    title: Contao Install-Tool
```

Comment:
Within Contao, the install tool is never referenced as "Contao Install-Tool", but always as "Contao install tool". Thus I would write it as "Contao Install Tool".

Reply:
So far, we have been using "Contao Install Tool" in English and "Contao Installtool" in German. I agree with @fritzmg that we should not introduce a third variant.

Reply:
Geändert in 078d2e186a65dfbd1fcdb9f28794a5a73d7fd359

Reply:
Dann muss es aber für die Deutsche Variante korrekterweise _Contao-Installtool_ sein, so ist es zumindest auch im Core: https://github.com/contao/contao/blob/feda301a54a68920d2b5be38f6a13b8641aa211b/installation-bundle/src/Resources/translations/messages.de.xlf#L6 (auch wenn mir persönlich das nicht so gefällt 😉)

Reply:
Ich persönlich wäre dafür in allen Fällen _Contao Install Tool_ zu schreiben, wir schreiben ja auch in allen Fällen _Contao Manager Plugin_ (und nicht bspw. _Contao-Managerplugin_ für die Deutsche Variante).